packages/vite/src/node/plugins/css.ts

```diff
@@ -1504,8 +1504,10 @@ function loadPreprocessor(
     return (loadedPreprocessors[lang] = _require(resolved))
   } catch (e) {
     if (e.code === 'MODULE_NOT_FOUND') {
+      const packageManager =
```

Comment:
I hope that the [`pkgFromUserAgent`](https://github.com/vitejs/vite/blob/13ac37df8e135fe6b4990fac6f149a36a19e5409/packages/create-vite/src/index.ts#L450) function can be reused in the same way as `import { pkgFromUserAgent } from '@vite/create-vite'` or `import { pkgFromUserAgent } from '@vite/shared'`, but currently, it is not exposed in that way.